### PR TITLE
Create block interactive template (discarded)

### PR DESCRIPTION
**Pull request by juanmaguitar**
## What?
Creation of the "create-block-interactive-template" and addition of "view.js" to the default "create-block"

## Why?
For the reasons explained [here](https://github.com/WordPress/gutenberg/pull/52508#issuecomment-1631105733)

### CODE REVIEW

**Review by ryanwelcher — APPROVED** [review]
The template code looks great and I love the addition of viewScript. Before we merge, not sure if we need another review in regards to this adding a new package.

**Comment by ryanwelcher** [comment]
cc @gziolo

**Comment by ryanwelcher** [comment]
@juanmaguitar it looks like we need to update the [Create Block test ](https://github.com/WordPress/gutenberg/blob/trunk/bin/test-create-block.sh#L58)to account for the new `view.js` file being added.

File: packages/create-block-interactive-template/block-templates/view.js.mustache
Comment (luisherranz) [inline packages/create-block-interactive-template/block-templates/view.js.mustache:1]: I see double-line breaks in this file. Is it on purpose?

File: packages/create-block-interactive-template/block-templates/view.js.mustache
Comment (luisherranz) [inline packages/create-block-interactive-template/block-templates/view.js.mustache:22]:
```suggestion
				context[ '{{namespace}}' ].isOpen = ! context[ '{{namespace}}' ].isOpen;
```

There's a missing space after the `!` (my fault).

**Review by luisherranz — CHANGES_REQUESTED** [review]
Hey, Juanma. The template code looks good to me but I don't understand why this PR is built on top of [a branch](https://github.com/WordPress/gutenberg/compare/trunk...create-block-interactive-template) that contains part of [the variant PR](https://github.com/WordPress/gutenberg/pull/52508#issuecomment-1631105733) instead of `trunk`. It's not clear to me what's going to be merged and why it will end up in a new branch. From the variant PR, we only need the `viewScript` property propagation.

Would it be worth switching the base of this PR to `trunk` so we can clearly see what's included? Thanks!!

**Comment by juanmaguitar** [comment]
I'm closing this PR in favor of this other PR https://github.com/WordPress/gutenberg/pull/52612

**Comment by juanmaguitar** [comment]
> Hey, Juanma. The template code looks good to me but I don't understand why this PR is built on top of [a branch](https://github.com/WordPress/gutenberg/compare/trunk...create-block-interactive-template) that contains part of [the variant PR](https://github.com/WordPress/gutenberg/pull/52508#issuecomment-1631105733) instead of `trunk`. It's not clear to me what's going to be merged and why it will end up in a new branch. From the variant PR, we only need the `viewScript` property propagation.
> 
> Would it be worth switching the base of this PR to `trunk` so we can clearly see what's included? Thanks!!

@luisherranz You're right. That PR was an attempt of using the same branch and the same PR fo these changes. 
But it's definitely more clear and easy to create a new branch and a new PR on top of `trunk` like https://github.com/WordPress/gutenberg/pull/52612

**Comment by luisherranz** [comment]
Thank you, Juanma. I've reviewed the other PR 🙂

**Pull request closed without merging**
